Formula/portable-openssl.rb

```diff
@@ -74,7 +83,7 @@ def install
       system "make"
       system "make", "test"
 
-      if build.with? "universal"
+      if build.with? "universal" && archs.length > 1
```

Comment:
Oh, good point.

I'm also rethinking my universal approach - even if "universal" usually doesn't include 64-bit archs on Tiger and Leopard, I may look at making the portable formulae build i386/ppc.